bgpd/bgp_io.c

```diff
@@ -114,7 +114,7 @@ void bgp_writes_on(struct peer *peer)
 {
 	while (
 	    !atomic_load_explicit(&bgp_io_thread_started, memory_order_seq_cst))
-		;
+		frr_pthread_yield();
```

Comment:
The addition of yield looks good, but there is zero reason for these flags to be using atomic operations.  They are always single writer and the tests are against true|false not a specific value.   The code should be changed to remove use of atomic ops on this (and the other) flag in the file.  @donaldsharp can you make this change as part of this patch?